src/main/resources/data.sql

```diff
@@ -121,4 +121,70 @@ VALUES
     (9, '2023-08-03 02:22:53', 50.00),
     (9, '2023-10-05 06:39:07', 20.00),
     (9, '2023-11-10 10:55:21', 80.00),
-    (9, '2023-12-12 15:11:35', 90.00);
\ No newline at end of file
+    (9, '2023-12-12 15:11:35', 90.00);
+
+INSERT INTO order_items (order_id, name, quantity)
+VALUES
+    (1, 'Pens', 2),
+    (1, 'Notebooks', 3),
+    (1, 'Paper Clips', 1),
```

Comment:
There is a single 'Paper Clips` order item name, while the rest use 'Paperclips' -- I'm not sure what the requirements are for this app.. maybe the order items are informational or decorative only. But it may be a good idea to add another table for `products` with order_items referencing it. However, this may just be a typo.